utils/protobuf/datawritemetadata.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package datawritemetadata;
+
+message IntegrationTimeMetadata {
+  uint32 experiment_id = 1;
+  string experiment_string = 2;
+  repeated SequenceType sequences = 3;
+  uint32 nave = 4;
+  uint32 last_seqn_num = 5;
+  bool scan_flag  = 6;
+  double integration_time  = 7;
+
+  message SequenceType {
+    repeated RXChan rxchannel = 1;
+    repeated uint32 blanks = 2;
+    TXDebugData tx_data = 3;
+
+    message RXChan {
+      uint32 slice_id = 1;
+      uint32 frang = 2;
+      uint32 nrang = 3;
+      repeated Beam beams = 4;
+      bool rx_only = 5;
+      uint32 pulse_len = 6;
+      uint32 tau_spacing = 7;
+      uint32 rxfreq = 8;
+      PulseTable ptab = 9;
+      bool acf = 10;
+      bool xcf = 11;
+      bool acfint = 12;
+      uint32 rsep = 13;
+      LagTable ltab = 14;
+      string comment_buffer = 15;
+      string interfacing = 16;
+      repeated uint32 rx_main_antennas = 17; // Main antennas received on for experiment slice.
+      repeated uint32 rx_intf_antennas = 18;
+
+      message Beam {
+      uint32 beamnum = 1;  // no channel phase data because beamforming already occurred
+      double beamazimuth = 2;
+      }
+
+      message PulseTable {  // length = length of sequence
+      repeated uint32 pulse_position = 1;
+      }
+
+      message LagTable {
+      repeated Lag lag = 1;
+
+        message Lag { // each Lag must have 2 pulse positions
```

Comment:
if each lag will always have 2 pulse positions, then is it better to have a repeated uint32 field, or just two separate uint32 values? 
Example: pulse_position_1, pulse_position_2

Reply:
Hmm. Not sure. Might have been a bit eaier to do it like that off the start, but its implemented and working like this now. thoughts @mardet987 ?

Reply:
I think I'm going to leave this - I don't think this is a problem